docs/import/create_score_asset.js

```diff
@@ -22,16 +22,20 @@ function countDamageAndBuildings(feature, damage, buildings) {
   const geoId = feature.get(geoidTag);
   const totalBuildings = ee.Algorithms.If(
       buildings.contains(geoId), buildings.get(geoId), ee.Number(0));
+  const rawIncomeValue = ee.String(feature.get(incomeTag));
+  const income = ee.Algorithms.If(
+      ee.Algorithms.IsEqual(rawIncomeValue, ee.String('250,000+')),
```

Comment:
maybe leave a TODO to handle any '<large number>+' similarly? I can imagine Census changes threshold

Reply:
Done in this PR